### PR TITLE
Modify contact form 

### LINKED ISSUE #318: UI Fix for Contact Us block
- [ ] Make the container background less dark as it hides the GitHub logo
- [ ] Turn Message input into textarea, allowing for a better visual appeal for long text inputs

image

**Contributor Information**
Please check if you are a contributor from:
- [ ] GSSoC-ext
- [ ] Hacktoberfest

### PR DESCRIPTION
**Pull request by mwango-phoenix**
# Title and Issue number 

Title : Contact form UI Modifications

Issue No. : 318

Close #318

# Image
image

# Checklist:

- [x] I have mentioned the issue number in my Pull Request.
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have created a helpful and easy to understand `README.md` if its a new page/tech stack
- [x] I have gone through the  `contributing.md` file before contributing

***Are you contributing under any Open-source programme?***

- [ ] I'm a GSSOC-EXT contributor
- [x] I'm a HACKTOBERFEST contributor

### CODE REVIEW

**Review by jeevan10017 — CHANGES_REQUESTED** [review]
Change the design

**Comment by mwango-phoenix** [comment]
> Change the design

in what way

**Comment by jeevan10017** [comment]
@mwango-phoenix, you changed from gradient colouring to normal, which does not look as good as before.

**Comment by mwango-phoenix** [comment]
> @mwango-phoenix, you changed from gradient colouring to normal, which does not look as good as before.

There was no gradient before as seen in the issue #318 . I checked the background multiple times and it was #121212, not sure what the linear gradient line was doing.
 I changed it since it hid the github logo

File: src/views/Contact.jsx [hunk 1]
```diff
@@ -23,7 +23,7 @@ const ContactForm = styled(Paper)(({ theme }) => ({
   padding: theme.spacing(4),
   maxWidth: "600px",
   margin: "auto",
-  backgroundColor: "linear-gradient(135deg, #1F1C2C 10%, #232526 100%)",
+  backgroundColor: "rgba(20, 20, 20, 0.8)",
```
Comment (jeevan10017) [inline src/views/Contact.jsx:26]: here

Comment (mwango-phoenix) [reply src/views/Contact.jsx:26]: I'm not sure what that looked like.  since what I got was the screenshot in #318 which had no gradient

Comment (jeevan10017) [reply src/views/Contact.jsx:26]: @mwango-phoenix, if you want to change the design further, you can. but the present is not looking good.

**Pull request closed without merging**